frame/support/procedural/src/storage/storage_info.rs

```diff
@@ -22,10 +22,6 @@ use quote::quote;
 use super::DeclStorageDefExt;
 
 pub fn impl_storage_info(def: &DeclStorageDefExt) -> TokenStream {
-	if !def.generate_storage_info {
-		return Default::default()
-	}
```

Comment:
I missed to remove this early return in the PR implementing storage info. the change is also available in https://github.com/paritytech/substrate/pull/9274